src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -137,6 +137,15 @@ def test_create_storage_account(self, resource_group, location):
         self.cmd('storage account check-name --name {}'.format(name),
                  checks=JMESPathCheck('nameAvailable', True))
 
+        large_file_name = self.create_random_name(prefix='cli', length=24)
+        self.cmd('storage account create -g {} -n {} --sku {} --enable-large-file-share'.format(
+            resource_group, large_file_name, 'Standard_LRS'))
```

Comment:
"rg" is predefined in ResourceGroupPreparer.
You don't need to type resource_group
e.g. `self.cmd('vm create -g {rg} -n {vm} --image UbuntuLTS')`

Reply:
I know. But in this function, it is already introduced as resource_group. I don't want to change the convention now.